x-pack/solutions/security/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -111,17 +106,24 @@ export const CellValue = memo(function RenderCellValue({
    */
 
   const finalData = useMemo(() => {
-    return (legacyAlert as TimelineNonEcsData[]).map((field) => {
-      if (['_id', '_index'].includes(field.field)) {
-        const newValue = field.value ?? '';
-        return {
-          field: field.field,
-          value: Array.isArray(newValue) ? newValue : [newValue],
-        };
-      } else {
-        return field;
-      }
-    });
+    const idPosition = legacyAlert.findIndex((field) => field.field === '_id');
+    const indexPosition = legacyAlert.findIndex((field) => field.field === '_index');
```

Comment:
`map` may actually be better here as it only iterates once. Need to do some testing on this as this is only really beneficial if `_id` and `_index` are both located early in the array. Will revert this change if it ends up being less performant after testing

Reply:
reverted